### DISCUSSION TITLE
My github page SSL

### DISCUSSION THREAD

**Discussion by MiguelNdeCarvalho** [community/community#21673] [Other Feature Feedback, Questions, & Ideas]
<p>Hey,</p>
<p>I can´t activate SSL for my page: <a href="http://www.miguelndecarvalho.me" rel="nofollow noopener">http://www.miguelndecarvalho.me</a> . Says me that: Enforce HTTPS — Unavailable for your site because a certificate has not yet been issued for your domain (<a href="http://www.miguelndecarvalho.me/" rel="nofollow noopener">www.miguelndecarvalho.me</a>) </p>
<p>Thank you,</p>
<p>Miguel de Carvalho</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/miguelndecarvalho">@miguelndecarvalho</a>,</p>
<p>Would you be able to take a look at <a href="https://help.github.com/articles/troubleshooting-custom-domains/#https-errors" rel="nofollow noopener">this article on HTTPS errors for custom domains using GitHub Pages</a> and confirm your domain is configured correctly? If it is, your best next step would be to <a href="https://github.com/contact" rel="nofollow noopener">contact GitHub Support</a> since they’d be best placed to look into this for you (forum members don’t have any special tooling to help out here)</p>
<p>​If there’s anything else I can help you with, let me know; so happy to help further!</p>
<p>Kindest regards,<br>
Matthew</p>

**Comment by MiguelNdeCarvalho** [comment]
<p>This is what I had</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e524ee87c7fd95722cc51ce8d549e6852307425c.png" data-download-href="/uploads/short-url/wH6sVQHojf2xZTEHAPS0l5WLpKc.png?dl=1" title="Opera Instantâneo_2018-05-23_172450_ap.www.namecheap.com.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33462772/181066288-60401846-df18-44a5-96da-59547d92b142.png" alt="Opera Instantâneo_2018-05-23_172450_ap.www.namecheap.com.png" data-base62-sha1="wH6sVQHojf2xZTEHAPS0l5WLpKc" width="690" height="180" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e524ee87c7fd95722cc51ce8d549e6852307425c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Opera Instantâneo_2018-05-23_172450_ap.www.namecheap.com.png</span><span class="informations">975×255 3.54 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by MiguelNdeCarvalho** [comment]
<p>What do you think?</p>

**Comment by ghost** [comment]
<p>Looks good to me <a class="mention" href="/miguelndecarvalho">@miguelndecarvalho</a>! I just checked your site at <a href="https://www.miguelndecarvalho.me/" rel="nofollow noopener">https://www.miguelndecarvalho.me/</a> and it seems to be going through to HTTPS successfully now!</p>
<p>​If there’s anything else I can help you with, let me know; so happy to help further!</p>
<p>Kindest regards,<br>
Matthew</p>

**Comment by coenraadhuman** [comment]
<p>Hi Matthew<br>
I seem to be experiencing the same problems with our project’s custom domain:</p>
<p><img src="https://user-images.githubusercontent.com/20205514/181066290-d217c3e0-6e17-4d64-bc94-7c8438bed448.png" alt="custom_domain_query.png" data-base62-sha1="aDtT5dlYvCDHKXtSlaUtysy8JJv" width="649" height="160"><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4360cd5649b6e14a4992008ea511a89631ee0d85.png" data-download-href="/uploads/short-url/9C3o3Hag7iVosdboQqjVwHaOvo9.png?dl=1" title="custom_domain_query_2.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/20205514/181066291-5b2db800-5d94-4c22-833e-effc7453f0a9.png" alt="custom_domain_query_2.png" data-base62-sha1="9C3o3Hag7iVosdboQqjVwHaOvo9" width="642" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/4360cd5649b6e14a4992008ea511a89631ee0d85_2_642x500.png, https://aws1.discourse-cdn.com/github/original/2X/4/4360cd5649b6e14a4992008ea511a89631ee0d85.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/4360cd5649b6e14a4992008ea511a89631ee0d85.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4360cd5649b6e14a4992008ea511a89631ee0d85_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">custom_domain_query_2.png</span><span class="informations">750×584 54.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
Went to the refered link in the discussion and made sure that all is fine. Resubmitted the custom domain in hope that it trigger the service on your side again.
<p>Is my configurations fine?<br>
It’s been roughly 6 hours, should we just wait?<br>
Is there a way to confirm that the certificate is being generated as speaking and being associated with the site?<br>
Or is there some sort of progress that can be tracked for this?</p>
<p>Kindest regards,<br>
Coenraad</p>

**Comment by coenraadhuman** [comment]
<p>Hi Matthew</p>
<p>The certificate generation is complete and associated.</p>
<p>Kindest regards,<br>
Coenraad</p>
